### PR TITLE
General report frequencies and callers...

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by dnil** [comment]
I kind of suspect this will not be enough, but I didn't have a good test case loaded on local, so testing on stage..

**Comment by dnil** [comment]
And the callers...
Screenshot 2022-04-01 at 11 36 51

 The callers construct is only present after the decoration. One could either add all callers individually to the variant filter constant, but that would sort of defeat the purpose of having the `CALLERS` constant. I opted for repopulating via CALLERS before decorating. Its a little complex as CodeFactor says. I can break it out into a function.

**Comment by northwestwitch** [comment]
> And the callers... Screenshot 2022-04-01 at 11 36 51
> 
> The callers construct is only present after the decoration. One could either add all callers individually to the variant filter constant, but that would sort of defeat the purpose of having the `CALLERS` constant. I opted for repopulating via CALLERS before decorating. Its a little complex as CodeFactor says. I can break it out into a function.

Nice! Perhaps a function that can be shared with variant page?

**Comment by dnil** [comment]
> Nice! Perhaps a function that can be shared with variant page? 

Yes, perhaps at least with the variant export functionality where it is even more similar. I'll check.

> Could also be a jinja macro

Mm, well, yes, that would be nice. As it is, we have macros for the variant types. They tend to be internally repetitive as well, but I dont think this is the time to touch that. But it would be nice to do it...

**Comment by northwestwitch** [comment]
> Mm, well, yes, that would be nice. As it is, we have macros for the variant types. They tend to be internally repetitive as well, but I dont think this is the time to touch that. But it would be nice to do it...

We can fix it later as well, yes!

**Comment by dnil** [comment]
Looking tentatively good. For a large var:
Screenshot 2022-04-01 at 12 00 22
And a smaller one with fewer panels:
Screenshot 2022-04-01 at 12 00 46

**Comment by northwestwitch** [comment]
Is there something missing here as well? 

image

**Comment by dnil** [comment]
>Is there something missing here as well? 
Indeed!

File: scout/constants/variant_tags.py [hunk 1]
```diff
@@ -37,6 +39,9 @@
     "category",
     "dismiss_variant",
     "variant_rank",
+    "sub_category",
+    "end_chrom",
```
Comment (northwestwitch) [inline scout/constants/variant_tags.py:43]: 👏🏻

Comment (dnil) [reply scout/constants/variant_tags.py:43]: Screenshot 2022-04-01 at 12 26 08

Comment (northwestwitch) [reply scout/constants/variant_tags.py:43]: Well done, let's merge before we see something else?

**Pull request merged**